### DISCUSSION TITLE
Idle Notifier type feature for "Cat is hungry"

### DISCUSSION THREAD

**Discussion by lupuleasa-core** [pieterjanbuntinx/kitten-tracker#30] [Ideas]
For all the people afk-ing, this would be massive if we got a windows notification when cat is hungry, similar to:
https://github.com/runelite/runelite/wiki/Idle-Notifier

thanks for listening, and thank you for a great plugin

**Comment by itsozone** [comment]
you could use the custom chat message notifier plugin that notifies when it sees keywords in chat messages.
